app/controllers/public/index.js

```diff
@@ -10,6 +10,10 @@ export default Controller.extend({
   { name: 'Speaker 4', organisation: 'Org 4', isFeatured: true, socialLinks: [{ name: 'twitter', url: '#' }], short_biography: 'Full-stack developer with experience in PHP, Python &amp; Ember.JS. Plays around with Kubernetes and Containers. Avid open source contributor.' },
   { name: 'Speaker 5', organisation: 'Org 5', isFeatured: false, socialLinks: [{ name: 'home', url: '#' }],  short_biography: 'Full-stack developer with experience in PHP, Python &amp; Ember.JS. Plays around with Kubernetes and Containers. Avid open source contributor.' }],
 
+  sponsors: [{ title: 'Gold Sponsors', sponsorList: [{ name: 'Sponsor 1', url: '#', logo: 'http://placehold.it/150x60' }, { name: 'Sponsor 2', url: '#', logo: 'http://placehold.it/150x60' }, { name: 'Sponsor 3', url: '#', logo: 'http://placehold.it/150x60' }] },
```

Comment:
@niranjan94 Okay so, should I write a custom function to group the sponsors by type, and sort the sponsors in each group by using `sort` computedProperty?